profiles/package.mask

```diff
@@ -29,6 +29,19 @@
 
 #--- END OF EXAMPLES ---
 
+# Michał Górny <mgorny@gentoo.org> (05 May 2017)
+# Old versions of sys-devel/gcc are no longer supported as the system
+# compiler. Packages are no longer tested against them, and may fail
+# to build. If you are still using an old GCC version as the system
+# compiler, you should upgrade sys-devel/gcc and switch gcc-config
+# to a new compiler ASAP. If you only need those GCC versions for
+# specific local uses, unmask them freely.
+<sys-devel/gcc-4.9
+<dev-python/pycuda-2016
+<dev-util/nvidia-cuda-sdk-7
+<dev-util/nvidia-cuda-toolkit-7
+net-wireless/cpyrit-cuda
```

Comment:
Can we please split this into two separate masks?

Further I would prefer to let the masking business of `sys-devel/gcc` to toolchain. :-)